installer_scripts/install-voyager-airgapped.sh

```diff
@@ -27,32 +25,27 @@ centos_yum_package_requirements=(
 )
 
 ubuntu_apt_package_requirements=(
+  "make|min|0"
   "sqlite3|min|0"
-  "gcc|min|0"
   "perl|min|0"
   "libdbi-perl|min|0"
   "libaio1|min|0"
-  "cpanminus|min|0"
   "libmysqlclient-dev|min|0"
+  "libmodule-build-perl|min|0"
```

Comment:
was this also just a miss ? or it's required now and not before?

Reply:
Yeah this was not required earlier.